<!-- This is an example launch file for how a ROS application could use a cordial_behavior -->
<launch>
   
	<arg name="use_behavior" default="true"/>
  	<group if="$(arg use_behavior)">
    		<node pkg="cordial_behavior" type="behaviors.py" name="behavior_node" output="screen"/>
  	</group>


</launch>
